### DISCUSSION TITLE
Difficulty applying for GitHub Education Plan with school email due to email delivery issue

### DISCUSSION THREAD

**Discussion by exzequel** [community/community#57614] [GitHub Education]
### Hi everyone,

I hope this message finds you well. I am writing to seek assistance with a persistent issue I am facing regarding the GitHub Education Plan application process.

I have made several attempts to apply for the GitHub Education Plan, providing all the necessary documentation to verify my eligibility. I have included my school ID, which clearly displays my name and the name of the institution. Additionally, I have submitted my Certificate of Registration (COR), which contains comprehensive details such as my class schedule and a recent date.

Unfortunately, despite my efforts, each application I have submitted using my personal email has been rejected. I have followed all the guidelines and ensured that the information I provided is accurate and valid. However, I have been unable to proceed further due to the rejection.

In light of this situation, I would like to apply again using my school email address  `umak.edu.ph` However, I have encountered a significant hurdle as my school email is currently experiencing an issue with receiving any emails from GitHub. This prevents me from completing the verification process and further delays my ability to access the benefits offered by the GitHub Education Plan.

I kindly request your guidance and support in resolving this matter. I am eager to utilize the resources and opportunities provided by the GitHub Education Plan to enhance my learning and collaboration in my academic pursuits.

If there are alternative methods or any additional steps I can take to successfully verify my eligibility, please provide me with the necessary instructions. I greatly appreciate your assistance in addressing this issue promptly.

Thank you for your attention and consideration. I look forward to your prompt response and resolution of the email delivery issue, allowing me to proceed with the GitHub Education Plan application.

**Comment by samus-aran** [comment]
Hi @exzequel 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. 
In this case it would be the Academic email(s) which has been associated with this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.
